apps/web/modules/videos/views/videos-single-view.tsx

```diff
@@ -42,8 +47,20 @@ export default function JoinCall(props: PageProps) {
       },
       url: meetingUrl,
       ...(typeof meetingPassword === "string" && { token: meetingPassword }),
+      customTrayButtons: {
+        transcription: {
+          label: "Enable Transcription",
+          tooltip: "Toggle Transcription",
+          iconPath: `${WEBAPP_URL}/sparkles.svg`,
+          iconPathDarkMode: `${WEBAPP_URL}/sparkles.svg`,
```

Comment:
we can only use https url here. ignore the icon not loading locally. it would load once this gets deployed